### PR TITLE
3 create html for homepage

### LINKED ISSUE #3: Create_HTML_for_homepage

### PR DESCRIPTION
**Pull request by Marcelin97**
close #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            content="Ohmyfood des menus gastronomiques à commander en ligne pour ne plus devoir attendre au restaurant"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./assets/CSS/tools.css">
+        <link rel="stylesheet" href="./assets/CSS/home.css" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&family=Shrikhand&display=swap"
+        rel="stylesheet"
+        />
```
Comment (Tutt-J) [inline index.html]: Load it before you own css

File: index.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    <!--head-->
+    <head>
+        <meta charset="UTF-8" />
+        <title>Oh my food</title>
+        <meta
+            name="description"
+            content="Ohmyfood des menus gastronomiques à commander en ligne pour ne plus devoir attendre au restaurant"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./assets/CSS/tools.css">
```
Comment (Tutt-J) [inline index.html]: No tools for this project

File: index.html [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        <link rel="stylesheet" href="./assets/CSS/home.css" />
```
Comment (Tutt-J) [inline index.html]: CSS but me in minuscule

File: index.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+                <i class="fas fa-map-marker-alt"></i><p>Paris, Belleville</p>
+            </div>
+        <!--end of page header-->
+        </header>
+        <main>
+        <!--main content-->
+            <section>
+                <h1>Réservez le menu qui vous convient</h1>
+                <p>Découvrez des restaurants d'exception, sélécctionnés par nos soins.</p>
+                <a href="#restaurants"></a><button>Explorer nos restaurants</button>
```
Comment (Tutt-J) [inline index.html]: no button here juste a

File: index.html [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+            </section>
+            <section>
+                <div>
```
Comment (Tutt-J) [inline index.html]: remove this div

File: index.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+        <!--end of page header-->
+        </header>
+        <main>
+        <!--main content-->
+            <section>
+                <h1>Réservez le menu qui vous convient</h1>
+                <p>Découvrez des restaurants d'exception, sélécctionnés par nos soins.</p>
+                <a href="#restaurants"></a><button>Explorer nos restaurants</button>
+            </section>
+            <section>
```
Comment (Tutt-J) [inline index.html]: add id

File: index.html [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+                >
+            </a>
+            <div>
+                <i class="fas fa-map-marker-alt"></i><p>Paris, Belleville</p>
+            </div>
+        <!--end of page header-->
+        </header>
+        <main>
+        <!--main content-->
+            <section>
```
Comment (Tutt-J) [inline index.html]: add id

File: index.html [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+                                Dégustez au restaurant
+                                </a>
+                            </li>
+                        </ol>
+                    </nav>
+                </div>
+            </section>
+            <section id="restaurants">
+                <h2>Restaurants</h2>
+                    <div>
```
Comment (Tutt-J) [inline index.html]: remove this div

File: index.html [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+                    </div>
+            </section>     
+        <!--main content-->
+        </main>
+        <footer>
+        <!--footer-->
+            <div>
+                <nav>
+                <h3>Ohmyfood</h3>
+                    <div>
```
Comment (Tutt-J) [inline index.html]: remove this div

File: index.html [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+                                    </a>
+                                </div>
+                        </article>
+                    </div>
+            </section>     
+        <!--main content-->
+        </main>
+        <footer>
+        <!--footer-->
+            <div>
```
Comment (Tutt-J) [inline index.html]: remove this div

File: package.json [hunk 11]
```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "autoprefixer": "^10.2.5",
     "css-loader": "^5.2.1",
+    "emmet": "^2.3.4",
```
Comment (Tutt-J) [inline package.json]: remove it

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html [hunk 12]
```diff
@@ -0,0 +1,174 @@
...
+            name="description"
+            content="Ohmyfood des menus gastronomiques à commander en ligne pour ne plus devoir attendre au restaurant"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&family=Shrikhand&display=swap"
+        rel="stylesheet"
+        />
+        <link rel="stylesheet" href="./assets/css/home.css" />
```
Comment (Tutt-J) [inline index.html]: Load it with import on typography.scss

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+            name="description"
+            content="Ohmyfood des menus gastronomiques à commander en ligne pour ne plus devoir attendre au restaurant"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Oh my food</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&family=Shrikhand&display=swap"
+        rel="stylesheet"
+        />
```
Comment (Tutt-J) [inline index.html]: You have to remove it, it's on typography

**Review by Tutt-J — CHANGES_REQUESTED** [review]
And remove test.html

File: index.html (hunk 2 above)
Comment (Marcelin97) [reply index.html]: I removed this links.

File: index.html (hunk 3 above)
Comment (Marcelin97) [reply index.html]: I wrote "css" in lowercase.

File: index.html (hunk 4 above)
Comment (Marcelin97) [reply index.html]: i removed the button tag.

File: index.html (hunk 5 above)
Comment (Marcelin97) [reply index.html]: i removed this div

File: index.html (hunk 6 above)
Comment (Marcelin97) [reply index.html]: I added "id" to my sections

File: index.html (hunk 7 above)
Comment (Marcelin97) [reply index.html]: I added "id" to my sections

File: index.html (hunk 8 above)
Comment (Marcelin97) [reply index.html]: i removed this div after h2

File: index.html (hunk 9 above)
Comment (Marcelin97) [reply index.html]: i removed this div after h3 in footer

File: index.html (hunk 10 above)
Comment (Marcelin97) [reply index.html]: i removed this div before h3 in footer

File: package.json (hunk 11 above)
Comment (Marcelin97) [reply package.json]: I uninstalled the plugin

File: index.html (hunk 12 above)
Comment (Marcelin97) [reply index.html]: i load it on typhography.scss

File: index.html (hunk 13 above)
Comment (Marcelin97) [reply index.html]: i removed this link because it is in typography.scss

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**